src/Raven.Client/Documents/Operations/Replication/PullReplicationAsSink.cs

```diff
@@ -12,6 +12,11 @@ public class PullReplicationAsSink : ExternalReplicationBase
         public string HubDefinitionName;
         public PullReplicationAsSink() { }
 
+        public PullReplicationMode Mode = PullReplicationMode.Outgoing;
+
+        public string[] Incoming;
+        public string[] Outgoing;
```

Comment:
`PullReplicationMode.Incoming / Outgoing` relates to the **direction** the data flows (To or From the hub)
'Incoming' & 'Outgoing' relate to **paths** that task is allowed to read from / write to

In order not to confuse, can we rename here to:
```
public string[] Incoming => WritePaths 
public string[] Outgoing => ReadPaths
```

Reply:
done